backend/src/BuiltinExecution/Libs/Bytes.fs

```diff
@@ -127,6 +64,45 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = NotYetImplemented
       previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "append" 0
+      typeParams = []
+      parameters = [ Param.make "bytes1" TBytes ""; Param.make "bytes2" TBytes "" ]
+      returnType = TBytes
+      description = "Returns the concatenation of <param bytes1> and <param bytes2>"
+      fn =
+        (function
+        | _, _, [ DBytes bytes1; DBytes bytes2 ] ->
+          bytes2 |> Array.append bytes1 |> DBytes |> Ply
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplemented
+      previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "containsSegment" 0
```

Comment:
What do you think about renaming `containsSegment` to just `contains`?

Reply:
I prefer `contains` as well. We use `List.contains`, right?

Reply:
I believe we call it `List.member`, we have` String.contains`